test/bedrock_Azure_mc_test.go

```diff
@@ -95,14 +161,14 @@ func TestIT_Bedrock_AzureMC_Test(t *testing.T) {
 	defer terraform.Destroy(t, tfOptions)
 	terraform.InitAndApply(t, tfOptions)
 
-	westCluster_out	:=	cluster_location1 + "-" + k8sName + "_kube_config"
-	eastCluster_out	:=	cluster_location2 + "-" + k8sName + "_kube_config"
-	centralCluster_out:=	cluster_location3 + "-" + k8sName + "_kube_config"
+	westCluster_out	:=	cluster_location1 + "_" + k8sName + "_kube_config"
```

Comment:
Is there code that's depending on this change?

Reply:
Don't believe so, just thought we'd switch to using underscores for consistency in our file names